src/main/resources/migreringer/V1.0_0024__oppgave_v2.sql

```diff
@@ -0,0 +1,38 @@
+CREATE TABLE if not exists BEHANDLING
+(
+     id                          VARCHAR(36)     NOT NULL PRIMARY KEY,
```

Comment:
vi bruker bigint for primary key i formidling og k9-sak også. Ser ut som det er bedre performance messig